server/src/main.js

```diff
@@ -26,64 +31,104 @@ const { User, Device } = models;
 
 const push = getPushEmitters({ models });
 
-const handlers = getHandlers({ models, creators, push });
+const handlers = getHandlers({ models, creators, push, pubsub });
 const resolvers = getResolvers(handlers);
 const schema = getSchema(resolvers);
 
-
 const app = express();
 
 app.use(cors());
 
-// `context` must be an object and can't be undefined when using connectors
+// Take incoming JTW and resolve user and device and return it. If not passed then use DEFAULT
+// user and device.
+const postJTWAuth = ({ id, device }) => new Promise((resolve) => {
```

Comment:
I like that you're thinking about function names but I think it should be named after what it does rather than where it is called.. perhaps you could name it `resolveUserDevice`?